### PR TITLE
Localazy 1.0.1

### PR DESCRIPTION
**Pull request by vaclavhodek**
### Reason for another PR

- We are sorry, we forgot to change step description that was coming from another step we used as a model. 

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/localazy/1.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  platform built by developers for developers \nto localize apps in the modern way. "
+website: https://localazy.com
+source_code_url: https://github.com/localazy/bitrise-step-localazy
+support_url: https://github.com/localazy/bitrise-step-localazy/issues
+published_at: 2020-11-26T14:12:21.200962895+01:00
+source:
+  git: https://github.com/localazy/bitrise-step-localazy.git
+  commit: 8190f3c85a295128b4a98fa72da1ca2f78fb29a0
+project_type_tags:
+- ios
```
Comment (koral--) [inline steps/localazy/1.0.1/step.yml:14]: If all projects types are supported then list may not be specified.

Comment (vaclavhodek) [reply steps/localazy/1.0.1/step.yml:14]: Should we remove it and create a new PR? I will remove it for subsequent releases.

Comment (vaclavhodek) [reply steps/localazy/1.0.1/step.yml:14]: I submitted another PR with this issue fixed.

**Comment by adborbas** [comment]
Closing this PR as discussed in https://github.com/bitrise-io/bitrise-steplib/pull/2778#issuecomment-737337468

**Pull request closed without merging**